setup.py

```diff
@@ -82,6 +82,7 @@
         "torch",
         "tqdm",
         "rich",
+        "patool",
```

Comment:
Is this still needed?

Reply:
Yes, unfortunately. The Spanish Passage Retrieval dataset is only available as a `.rar` file and there is no specific copyright license in place. We tried contacting one of the authors to ask for permission to republish the dataset on HuggingFace, but we haven't heard back yet.

Reply:
We ended up getting a response from the original creators of the Spanish Passage Retrieval dataset and a go-ahead to host the dataset on HF. So, I've removed the `patool` requirement since it's no longer needed.